README.md

```diff
@@ -11,25 +12,155 @@ This operator is in charge to configure the idp client service.
 
 `make functional-test-full`
 
-# Deploy the operator
+# Installing
 
-1. Login on your hub
-2. From the idp-mgmt-operator: `make deploy`
+## Prereqs
+You must meet the following requirements:
+- OpenShift Container Platform (OCP) 4.6+
+- `oc` (ver. 4.6+)
 
-# Example
+## Ensure you are logged in to the correct OCP hub
+```bash
+oc cluster-info
+```
+
+## Setup Hub to use a signed certificate
+This is required for the Dex server we will be using under the covers to authenticate
+the managed clusters OpenID OAuth requests.  The quickest way to do this is using a tool from
+https://github.com/open-cluster-management/sre-tools/wiki/ACM---Day-1#add-an-acme-certificate.  Here is a summary of the commands you need to run:
+
+```bash
+export AWS_ACCESS_KEY={your AWS Key}
+export AWS_SECRET_ACCESS_KEY={your AWS Secret Key}
+export MY_EMAIL_ADDR={your email address}
+
+cd /tmp
+git clone https://github.com/acmesh-official/acme.sh.git
+cd acme.sh
+
+export API=$(oc whoami --show-server | cut -f 2 -d ':' | cut -f 3 -d '/' | sed 's/-api././')
+export WILDCARD=$(oc get ingresscontroller default -n openshift-ingress-operator -o jsonpath='{.status.domain}')
+
+
+./acme.sh --register-account -m ${MY_EMAIL_ADDR}
+
+
+./acme.sh  --issue   --dns dns_aws -d ${API} -d "*.${WILDCARD}"
+
+pushd ${PWD}
+cd ${HOME}/.acme.sh/${API}
+
+oc create secret tls router-certs --cert=fullchain.cer --key=${API}.key -n openshift-ingress
 
-An example of the Authrealm CR is available [here](test/config/example/github-authrealm.yaml)
+oc patch ingresscontroller default -n openshift-ingress-operator --type=merge --patch='{"spec": { "defaultCertificate": { "name": "router-certs" } } }'
 
-You can do a `kubectl apply -f test/config/example/github-authrealm.yaml` to set that example
-and then delete the example with `kubectl delete -f test/config/example/github-authrealm.yaml`
+popd
+```  
 
-This example creates a placement with a matchlabel `authdeployment: east` and so you have to add that label in the managedcluster you want to be managed by the authrealm AND add a label `cluster.open-cluster-management.io/clusterset: cluster-sample` in it.
+After running these commands, various pods on the OCP hub will restart in order to use the new certificate.  Wait a few minutes for all the pods to restart.  The OCP UI Overview page can be used to check the overall health of OCP.
 
-```yaml
-    authdeployment: east
-    cluster.open-cluster-management.io/clusterset: clusterset-sample
+NOTE: To use the ACME certificate process, you must have Amazon AWS credentials to allow a Route53 domain to
+be added for certificate verification during creation.
+
+
+## Setup GitHub OAuth
+Use GitHub as the OAuth provider.
+1. On github.com, go to Settings > Developer Settings > OAuth Apps
+(The shortcut is https://github.com/settings/developers)
+
+2. Add a `New OAuth App`.  Copy the `Client ID` and `Client Secret` values.  You will need them later on.
```

Comment:
Should it be at the org level or user level?

Reply:
I think yesterday we said it did not need to be at org level, even though we at first thought so.  I will leave at user level for now and verify it works.  Will update if needed